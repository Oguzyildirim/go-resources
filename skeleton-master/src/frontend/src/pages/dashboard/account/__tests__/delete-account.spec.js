import { mount } from '@vue/test-utils'
import Vuex from 'vuex'
import VueRouter from 'vue-router'
import initData from '@@/globals/init-data'
import Page from '../delete-account'

describe('delete-account.vue', () => {
  let store

  beforeEach(() => {
    const actions = {
      'page/title': jest.fn(),
      'loading/start': jest.fn(),
      'loading/finish': jest.fn()
    }
    const getters = {
      'user/user': () => initData().user
    }
    store = new Vuex.Store({
      actions,
      getters,
      state: {
        loading: { status: false }
      }
    })
  })

  it('should match snapshot', () => {
    const router = new VueRouter({})
    const w = mount(Page, {
      store,
      router,
      stubs: ['router-link', 'router-view']
    })

    expect(w.html()).toMatchSnapshot()
    w.destroy()
  })
})
