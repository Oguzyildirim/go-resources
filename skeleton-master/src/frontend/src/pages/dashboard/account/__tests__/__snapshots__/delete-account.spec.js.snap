// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`delete-account.vue should match snapshot 1`] = `
<section class="container container--fluid">
  <div class="row justify-center">
    <div class="v-card--material pa-3 pa-6 v-card v-sheet theme--light v-card--material--has-heading" style="min-width: 50%;" min-width="50%">
      <div class="d-flex grow flex-wrap">
        <div class="text-start v-card--material__heading mb-n6 v-sheet theme--dark elevation-6 info pa-7" style="width: 100%;">
          <div class="display-2 font-weight-light">
            Delete Account
          </div>
          <div class="subtitle-1 font-weight-light">
            Once the account is delete it cannot be restored
          </div>
          <!---->
          <!---->
        </div>
        <!---->
      </div> <span> <!----> <!----></span>
      <div class="container">
        <div class="text-center col col-12"><button type="submit" class="mr-0 v-btn v-btn--contained theme--light v-size--default error"><span class="v-btn__content">
            Delete Account
          </span></button></div>
      </div>
      <!---->
    </div>
  </div>
  <div role="dialog" class="v-dialog__container" style="z-index: 200;">
    <!---->
  </div>
</section>
`;
